baton/handler.go

```diff
@@ -14,6 +15,8 @@ func Handler() *mux.Router {
 	return m
 }
 
+var i = 0
```

Comment:
Several things are not set up to be ready for a load balancer. When/if we get to the point that a load balancer makes sense, I'll have to change it. 

I wouldn't expect we would need a load balancer any time in the immediate future. Also, load balancing websockets such that they would switch which machine they are messaging is probably not ideal. 

tldr: I'm not expecting us to use loadbalancing soon; if we do it so that websockets only connect to one machine at a time, we will be fine anyways.